src/scss/_brand.scss

```diff
@@ -0,0 +1,141 @@
+////
+/// @group o-label
+/// @link http://registry.origami.ft.com/components/o-label
+////
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oLabelGet($variables, $from: null) {
+	@return oBrandGet($component: 'o-label', $variables: $variables, $from: $from);
+}
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oLabelSupports($variant) {
+	@return oBrandSupportsVariant($component: 'o-label', $variant: $variant);
+}
+
+// Define shared brand configurations
+$_label-border-width: 1px;
+$_o-label-shared-brand-config: (
+	default-font-size: nth(oTypographyGetScale(0), 1) * 1px,
+	default-border-width: $_label-border-width,
+	default-padding: (oTypographySpacingSize(1) - $_label-border-width) (oTypographySpacingSize(2) - $_label-border-width),
```

Comment:
Brand variants mean we don't need to prefix with `default-`. If we open up o-brand or start outputting CSS custom properties, we'll be able to document fewer variables. :)